apps/src/templates/testPages/WebLabTest.jsx

```diff
@@ -22,14 +24,58 @@ class WebLabTest extends Component {
     this.state = {
       [STATUS_CODE_PROJECTS]: Status.WAITING,
       [STATUS_COMPUTING_IN_THE_CORE]: Status.WAITING,
+      [STATUS_BRAMBLE_MOUNTABLE]: Status.WAITING,
       renderCallToAction: false
     };
   }
 
-  promisifiedTestImageAccess = ({url, status}) =>
+  componentDidMount = () => {
+    window.addEventListener('message', this.updateBrambleStatus);
+  };
+
+  componentWillUnmount = () => {
+    window.removeEventListener('message', this.updateBrambleStatus);
+  };
+
+  updateBrambleStatus = (event = {}) => {
+    let message;
+    try {
+      message = JSON.parse(event.data);
+    } catch (err) {
+      console.log(err);
+    }
+
+    if (
+      event.origin === 'http://localhost-studio.code.org:3000' &&
```

Comment:
is this test code? we don't want to check against localhost in production

Reply:
Oh whoops, I think I had changed this to reference `studioUrl` but it looks like it got swapped back, thanks!

Reply:
I think this line of code is out-of-date: https://github.com/code-dot-org/code-dot-org/pull/49752/files#diff-3a139ce9c6bc93c7143290eb0f44976e2fd9737aef8af8a68ac5e7c874df9758R57